### DISCUSSION TITLE
How to see `tags` in GeoTiff?

### DISCUSSION THREAD

**Discussion by iseong83** [corteva/rioxarray#573] [Q&A] [answered]
How can I see the tags using `open_rasterio` from GeoTIFF? 
I am adding an additional tag to GeoTIFF and I can see this using rasterio:
<img width="599" alt="image" src="https://user-images.githubusercontent.com/44355378/188735836-6ceebd94-b1d4-4d34-9b80-362f73133e92.png">

But, I can not see the same information from xr.Dataset.
<img width="720" alt="image" src="https://user-images.githubusercontent.com/44355378/188730963-48eaf9f4-475f-4074-862e-eddc6ed1dac9.png">
Is there a way to see `tags`?

**Comment by snowman2** [comment, marked as answer]
See #296

**Reply by snowman2** [reply]
This may resolve your issue: https://github.com/corteva/rioxarray/pull/575

**Reply by iseong83** [reply]
Awesome! Thank you so much!

**Reply by snowman2** [reply]
See #600
